dspace-api/src/main/java/org/dspace/util/SolrImportExport.java

```diff
@@ -166,7 +174,7 @@ else if ("export".equals(action))
                     {
                         String solrUrl = makeSolrUrl(indexName);
                         String timeField = makeTimeField(indexName);
-                        exportIndex(indexName, exportDir, solrUrl, timeField, lastValue);
+						exportIndex(indexName, exportDir, solrUrl, timeField, lastValue, line.hasOption(OVERWRITE_OPTION));
```

Comment:
While not a strict requirement, @terrywbrady, the alignment of your code changes is rather odd in your recent PRs (not just this one, but others as well). You may want to look closer at your IDE, as it almost looks like any code change you make replaces spaces with tabs. I, personally, tend to lean towards spaces as the alignment is more consistent across OSes.  Either is OK, but we should work to ensure the lines align visually when they get into GitHub.